docs/service-options.md

```diff
@@ -20,7 +20,7 @@ module.exports.options = {
 
 The values supported in this object are the supported keys and value for the Fastify server instance
 that you can find at this [link][fastify-server-options]; with the exception of the `logger` parameter.  
-Instead you can customize the `pino` instance via the `logLevel` key and you can modify the redaction rules
+Instead you can customize the `pino` instance via the `logLevel` and `customLogLevel` keys, and you can modify the redaction rules
```

Comment:
can you put an example of how the customLogLevel should be configured here?

Reply:
Yes, I just added it in the snippet above.